File: JAVA/Preparation/Palindrome.java
// Program to Check number is Palindrome or not

import java.util.*;
public class Palindrome {
    public static void main(String[] args) {
        Scanner ar = new Scanner(System.in);
        System.out.print("\n Enter any Number : ");
        int num = ar.nextInt();
        ar.close();
        int sum=0, rem;
        int temp = num;
        while(num>0) {
            rem = num % 10;
            sum = (sum * 10) + rem;
            num = num / 10;
        }
        if (temp == sum) {
            System.out.println("\n "+temp+" is a Palindrome");
        }
        else {
            System.out.println("\n "+temp+" is NOT a Palindrome");
        }
    }
}

File: C-Programming/Practice/Basic C/palindrome.c
//program to print number is palindrome or not
#include<stdio.h>
int main()
{
    int num,reverse=0,copy_num;
    printf("\n Enter any Number : ");
    scanf("%d",&num);
    copy_num = num;
    while (num!=0)
    {
        reverse = (reverse * 10) + (num % 10);
        num = num / 10;
    }
    printf("\n Reverse is : %d",reverse);
    if (copy_num == reverse)
    {
        printf("\n %d is a Palindrome Number",copy_num);
    }
    else
    {
        printf("\n %d is NOT Palindrome Number",copy_num);
    }
    return 0;
}